pkg/solana/chainwriter/chain_writer.go

```diff
@@ -167,6 +167,14 @@ func GetAddresses(ctx context.Context, args any, accounts []Lookup, derivedTable
 	return addresses, nil
 }
 
+// These errors are ignorable if the lookup is optional.
+func isIgnorableError(err error) bool {
+	return err == nil ||
```

Comment:
Wouldn't this always be false because of the condition above? Since we're checking err != nil before?`accountConfig.Optional && err != nil && isIgnorableError(err)`

Reply:
Yea true, I'll remove